views/vote/confirmRegistration.pug

```diff
@@ -0,0 +1,29 @@
+extends ../base
+
+block variables
+  -var title = __('Register to vote')
+  -var voterCitizen = !hasData(data, 'vote.voterCitizen') ? '' : data.vote.voterCitizen
+  -var voterConsent = !hasData(data, 'vote.voterConsent') ? '' : data.vote.voterConsent
+
+block content
+
+  h1 #{title}
+  
+  div
+    p #{__('The Canada Elections Act gives rules about elections and voter information. Provincial and territorial governments, members of Parliament, registered political parties, and candidates at election time can have your voter information.')}
+
+    p #{__('If you complete this section, you will be registered to vote until you file your next tax return.')}
+    
+  form.cra-form(method='post')
+    .multiple-choice.multiple-choice--checkboxes
```

Comment:
Let's wrap this in a [fieldset](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset) like our other radios mixin: https://github.com/cds-snc/cra-claim-tax-benefits/blob/master/views/_includes/radiosMixin.pug

Then we can remove the `label` below and use a `legend` instead.